db/iscn.go

```diff
@@ -123,42 +127,84 @@ func QueryISCNList(pool *pgxpool.Pool, pagination Pagination) ([]iscnTypes.Query
 	defer cancel()
 
 	sql := fmt.Sprintf(`
-		SELECT tx #> '{"tx", "body", "messages", 0, "record"}' as data, events, tx #> '{"timestamp"}'
+		SELECT id, tx #> '{"tx", "body", "messages", 0, "record"}' as data, events, tx #> '{"timestamp"}'
 		FROM txs
-		WHERE events @> '{"message.module=\"iscn\""}'
+		WHERE events && '{"message.action=\"create_iscn_record\"",
+			"message.action=\"update_iscn_record\"",
+			"message.action=\"/likechain.iscn.MsgCreateIscnRecord\"",
+			"message.action=\"/likechain.iscn.MsgUpdateIscnRecord\""
+		}'
```

Comment:
I think `message.module=iscn` would work in both cases?
https://mainnet-node.like.co/cosmos/tx/v1beta1/txs?events=message.module=%27iscn%27&order_by=ORDER_BY_DESC